src/app/modules/item/pages/item-edit-advanced-parameters/item-edit-advanced-parameters.component.ts

```diff
@@ -90,23 +81,41 @@ export class ItemEditAdvancedParametersComponent implements OnInit, OnChanges {
     this.parentForm?.get('duration')?.updateValueAndValidity();
   }
 
+  onEnteringTimeMinEnabledChange(event: boolean): void {
+    if (!event) {
+      return;
+    }
+
+    this.parentForm?.get('entering_time_min')?.patchValue(
+      new Date()
+    );
```

Comment:
So, just switching it off and on directly clear the previous value. This is not what we want. Should set to now only if it equaled DEFAULT_ENTERING_TIME_MIN.

Reply:
Fixed